src/codecs.js

```diff
@@ -1,3 +1,15 @@
+import window from 'global/window';
+
+const mp4Regex = RegExp('^(av0?1|avc0?[1234]|vp0?9|flac|opus|mp3|mp4a|mp4v)');
+const webmRegex = RegExp('^(vp0?[89]|av0?1|opus|vorbis)');
+const oggRegex = RegExp('^(vp0?[89]|theora|flac|opus|vorbis)');
+
+const videoCodecRegex = RegExp('^(av0?1|avc0?[1234]|vp0?[89]|hvc1|hev1|theora|mp4v)');
+const audioCodecRegex = RegExp('^(mp4a|flac|vorbis|opus|ac-[34]|ec-3|alac|mp3)');
+
+const muxerVideoRegex = RegExp('^(avc0?1)');
```

Comment:
Should this and the regex for audio below exist in mux.js instead of here? I am thinking that it should.